### ISSUE TITLE
VPC lookup function is not configured properly

### ISSUE THREAD

**Issue by Jeevan-J** [mongodb-partners/mongodb_atlas_as_aws_bedrock_knowledge_base#1]
While running `cdk bootstrap`, the default VPC ID and VPC CIDR are shown in the logs.

```shell
> cdk bootstrap
VPC ID: vpc-12345
VPC CIDR: 1.2.3.4/5
[]
Ports: 1024,1025,1026
Security Group ID: ${Token[TOKEN.21]}
Load Balancer ARN: ${Token[TOKEN.34]}
VPC Endpoint IPs: 10.0.0.7
Listener ARN for port 1024: ${Token[TOKEN.38]}
IP Targets for port 1024: [{"ipAddress":"10.0.0.7","port":1024}]
Listener ARN for port 1025: ${Token[TOKEN.57]}
IP Targets for port 1025: [{"ipAddress":"10.0.0.7","port":1025}]
Listener ARN for port 1026: ${Token[TOKEN.76]}
IP Targets for port 1026: [{"ipAddress":"10.0.0.7","port":1026}]
```

After debugging, I found that the following line `lib\mongodb_atlas_as_aws_bedrock_knowledge_base-stack.ts` is not configured properly as per [CDK Lib docs](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ec2.VpcLookupOptions.html)
```ts
const vpc = ec2.Vpc.fromLookup(this, vpcId, { isDefault: false });`
```

This line has to be modified as the following to make this work.
```ts
const vpc = ec2.Vpc.fromLookup(this, 'Vpc', { isDefault: false, vpcId: vpcId });`
```

Basically this filters the VPC using VPC ID.
